hardhat.config.ts

```diff
@@ -26,6 +27,17 @@ task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
   }
 });
 
+task("transfer", "Prints the list of accounts", async (taskArgs, hre) => {
```

Comment:
Wrong name, probably from copy&paste

Reply:
this task is removed since it was a helper task to move funds from my HD wallet to deployer account.